x/issuance/types/params.go

```diff
@@ -0,0 +1,131 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// Parameter keys and default values
+var (
+	KeyAssets         = []byte("Assets")
+	DefaultAssets     = Assets{}
+	ModuleAccountName = ModuleName
+)
+
+// Params governance parameters for the issuance module
+type Params struct {
+	Assets Assets `json:"assets" yaml:"assets"`
```

Comment:
ok, so I think Assets should be governance params. They should be set to the issuance module store and defined on genesis. If a new asset needs to be added/removed you can create a custom gov proposal type for both operations and create a custom handler to add/delete assets from the store.

This way you have fine-grained control over the validation of the assets, eg validate that users cannot add and delete assets on the same param change proposal. Also, param change proposals don't allow you to validate the new value with the previously stored ones.

Reply:
Assets already are goverance params because they are part of the `Params` for the module, so we can use `param-change` proposals to add/remove/edit assets. I think this is sufficient for governance for now, can expand if necessary as scope becomes better defined.